config/initializers/paper_trail.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Disabling creation of the new versions for each model declared under the key :versioning_disable_for_models in config/yeti_web.yml
+# Access to paper_trail methods like `object.versions` still present
+Rails.configuration.yeti_web['versioning_disable_for_models']&.each do |model_name|
+  normalized_model_name = model_name&.gsub('/', '::')&.safe_constantize
+  PaperTrail.request.disable_model(normalized_model_name) if normalized_model_name.present?
```

Comment:
do not check that `normalized_model_name` is present, when configuration has invalid model classes then the error should be raised. use constantize instead of safe_constantize